orchestra/specs/mistral/v2/workflows.py

```diff
@@ -47,7 +47,8 @@ class WorkflowSpec(base.Spec):
             'input': types.UNIQUE_STRING_OR_ONE_KEY_DICT_LIST,
             'output': types.NONEMPTY_DICT,
             'task-defaults': tasks.TaskDefaultsSpec,
-            'tasks': tasks.TaskMappingSpec
+            'tasks': tasks.TaskMappingSpec,
+            'type': types.WORKFLOW_TYPE
```

Comment:
Since the workflow type is specific to the Mistral DSL, can we move the WORKFLOW_TYPE to a different module (i.e. `orchestra/specs/mistral/types.py`)?  The Orchestra DSL is by default direct only and there's no current plan to add reverse. In our experience, any reverse workflow can be described as a direct workflow. But we are open if users can identify a use case where it is not possible.

Reply:
Also, can you move the `type` attribute to the top of `properties`?